modules/space/rendering/renderableeclipsecone.cpp

```diff
@@ -127,6 +123,41 @@ namespace {
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
+
```

Comment:
Nitpicky, but do we want the extra newlines around this function?

Reply:
Nope, we don't :D